cmd/triggers/create.go

```diff
@@ -116,7 +116,7 @@ func runCreateCommand(clients *shared.ClientFactory, cmd *cobra.Command) error {
 		app = _app
 	}
 
-	err = validateCreateCmdFlags(clients, &createFlags)
+	err = validateCreateCmdFlags(ctx, clients, &createFlags)
```

Comment:
note: For the most part, this PR updates 2 functions to accept `ctx context.Context`. So, you'll see a lot of changes that just add `ctx` as an argument.